samples/peripheral/802154_sniffer/README.rst

```diff
@@ -90,6 +90,19 @@ The ``sleep`` command disables the radio and ends the receive process.
 
       sleep
 
+bootloader - reboot the device to the bootloader
+================================================
+
+The ``bootloader`` command reboots the device without skipping the bootloader.
+
+   .. parsed-literal::
+      :class: highlight
+
+      sleep
```

Comment:
Should be `bootloader` here. Also if the bootloader prints something or blinks some LED, it would be good to mention that or link to some documentation page.

Reply:
right. Added a line about red LED pulsing after jumping to bootloader